xml/Microsoft.VisualBasic.CompilerServices/Conversions.xml

```diff
@@ -1444,9 +1444,9 @@
         <Parameter Name="Value" Type="System.Double" />
       </Parameters>
       <Docs>
-        <param name="Value">The <see langword="Double" /> value to convert.</param>
+        <param name="Value">The <see cref="T:System.Double" /> value to convert.</param>
         <summary>Converts a <see cref="T:System.Double" /> value to a <see cref="T:System.String" /> value.</summary>
-        <returns>The <see langword="String" /> representation of the <see langword="Double" /> value.</returns>
+        <returns>The <see langword="String" /> representation of the <see cref="T:System.Double" /> value.</returns>
```

Comment:
You already have the type of the return value showing up next to it, so you can use natural language here:
```suggestion
        <returns>The string representation of the <see cref="T:System.Double" /> value.</returns>
```